ibridgesgui/search.py

```diff
@@ -101,8 +98,9 @@ def load_results(self, results):
         table_data = []  # (Path, Name, Size, Checksum, created, modified)
         for result in results:
             if "DATA_NAME" in result:
-                obj = IrodsPath(self.session,
-                                result["COLL_NAME"] + "/" + result["DATA_NAME"]).dataobject
+                obj = IrodsPath(
```

Comment:
Instead of the "/" and "+", you can just use a comma?

Reply:
Indeed!